<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Exclusive Choose OS</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="WindowscheckBox">
          <property name="text">
           <string>Windows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="LinuxcheckBox">
          <property name="text">
           <string>Linux</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="MaccheckBox">
          <property name="text">
           <string>Mac</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Non-Exclusive Choose Drink</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="juicecheckBox">
          <property name="text">
           <string>Juice</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="beercheckBox">
          <property name="text">
           <string>Beer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="winecheckBox">
          <property name="text">
           <string>Wine</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Choose Answer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="AradioButton">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="BradioButton">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="CradioButton">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="enterData">
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setButton">
     <property name="text">
      <string>Set State</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
